hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -71,9 +71,10 @@ For details on the options for application interface commands please see the [CS
 Once the CSRNG instance has been instantiated, and the `generate` and `reseed` commands have been loaded, auto request mode can be entered by programming the [`CTRL`](registers.md#ctrl) register with `EDN_ENABLE` and `AUTO_REQ_MODE` fields are enabled.
 Note that if BOOT_REQ_MODE is asserted the state machine will enter boot-time request mode, even if AUTO_REQ_MODE is asserted.
 
-To issue any new commands other than those stored in the generate or reseed FIFOs, it is important to disable auto request mode, by deasserting the `AUTO_REQ_MODE` field in the [`CTRL`](registers.md#ctrl) register.
+To issue any new commands other than those stored in the generate or reseed FIFOs, it is important to disable auto request mode, by deasserting the `AUTO_REQ_MODE` or the `EDN_ENABLE` field in the [`CTRL`](registers.md#ctrl) register.
```

Comment:
I'd consider splitting this sentence, which might make it easier to follow. Maybe "... to disable the auto request mode. Do this by configuring the CTRL register to deassert the AUTO_REQ_MODE or EDN_ENABLE field."

(I struggled to phrase this in a way that didn't sound like AUTO_REQ_MODE was a mode, but EDN_ENABLE was a field!)

Reply:
Agreed, thanks! I'll change it in my next push.